scripts/deployment/l2/deploy_05_07_change_managers_and_permissions.js

```diff
@@ -38,6 +38,10 @@ async function main() {
         networkURL = "https://rpc.gnosischain.com";
     } else if (providerName === "chiado") {
         networkURL = "https://rpc.chiadochain.net";
+    } else if (providerName === "arbitrumOne") {
```

Comment:
note that we need to change `serviceManagerAddress = parsedData.serviceManagerAddress;` in line 16 with `serviceManagerAddress = parsedData.serviceManagerTokenAddress;`

Reply:
All is correct, I just think we should not modify original scripts. Maybe just have a separate script that does the setup for all the changed fields accounting for the fact that the manager token is used from the very start.

Reply:
Adjusted all of them, changed the deployment workflow description